packages/npm-dependencies-validation/package.json

```diff
@@ -15,17 +15,20 @@
   "files": [
     ".reuse",
     "LICENSES",
-    "dist/src/api.d.ts",
-    "dist/src/api.js"
+    "README.md",
+    "CHANGELOG.md",
+    "dist/src/**/*.js",
+    "dist/src/**/*.js.map",
```

Comment:
including `.js.map` and `src/*.ts` file enables consumers to debug
the TypeScript sources directly.